Packages/Nakama/Runtime/UnityWebRequestAdapter.cs

```diff
@@ -32,6 +32,7 @@ namespace Nakama
     /// </remarks>
     public class UnityWebRequestAdapter : MonoBehaviour, IHttpAdapter
     {
+    #if UNITY_2020_2_OR_NEWER

```

Comment:
You don't need to put this around all the members in the class. Only the two if  statements where you are actually doing the `if` statements.

It would probably be best to have an `IsConnectionError` method and put the `#if` directive there.

Reply:
Thank you for your response again. I put all the error handling in a method as you said. I assume it will work now?